src/Model/collections.ts

```diff
@@ -152,6 +168,19 @@ Model.prototype.getOrCreateDoc = function(collectionName, id, data) {
   return collection.getOrCreateDoc(id, data);
 };
 
+Model.prototype.getOrDefault = function<S>(subpath: Path, defaultValue: S) {
```

Comment:
Mentioning for discussion - as this is now, it won't support a 1-arg call like `getOrDefault(defaultValue)`, though there is the workaround of doing `getOrDefault('', defaultValue)`.

We can always add it later if needed and we don't want to add it now.

Reply:
looks like the intent is as a fallback in case the subpath fails? . which would make sense then to keep the structure... I'd argue that providing blank subpath or defaultValue raise error no? They are both required?

Reply:
Eric and I discussed ofline and agree the no-subpath case is unlikely, so leaving as is and see if any use comes up.